src/Arcus.Scripting.Tests.Unit/Arcus.Scripting.ApiManagement.tests.ps1

```diff
@@ -307,14 +307,26 @@ InModuleScope Arcus.Scripting.ApiManagement {
                 $serviceName = "shopping-API-management"
                 $context = New-Object -TypeName Microsoft.Azure.Commands.ApiManagement.ServiceManagement.Models.PsApiManagementContext
 
+                Mock Get-AzApiManagementApi {
+                    $Context | Should -Be $context
+                    $ApiId | Should -Be "echo-api"
+                    return $null } -Verifiable
```

Comment:
Should we then here also verify on the paremeter? Like, `-ParameterFilter { $ApiId -eq "echo-api" }`, plus include it in the assertion?

Reply:
I changed the verify and assertion accordingly